source/connect/connections.txt

```diff
@@ -0,0 +1,81 @@
+.. _compass-sidebar:
+
+===============
+Compass Sidebar
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   .. include:: /includes/multiple-connections.rst
+
+
+The Compass sidebar on the left of the Compass window
+contains the following options:
+
+- :icon-lg:`Plus` icon to add new connections
+- :icon-fa5:`ellipsis-h` icon to import and export data
+- Search bar to search connections
+- :guilabel:`Connections` list
+
+.. figure:: /images/compass/connections-sidebar.png
+   :figwidth: 750px
+   :alt: Connections Sidebar
+
+Connections List
+----------------
+
+The :guilabel:`Connections` list in the sidebar contains your
+:ref:`favorite <favorite-connections>` and saved connections. Favorite
+connections have a :icon-fa5:`star` next to the connection name.  The
```

Comment:
maybe we can add that favorites will always appear at the top of the connections list?

Reply:
Done!